python/mlc_chat/interface/compiler_flags.py

```diff
@@ -119,6 +119,15 @@ def __post_init__(self):
                     self.sliding_window_size,
                 )
                 self.prefill_chunk_size = self.sliding_window_size
+            elif self.prefill_chunk_size > self.sliding_window_size:
+                logger.info(
+                    "Lower %s (%d) to %s (%d) as the upper bound",
+                    bold("prefill_chunk_size"),
+                    self.prefill_chunk_size,
+                    bold("sliding_window_size"),
+                    self.sliding_window_size,
+                )
+                self.prefill_chunk_size = self.sliding_window_size
```

Comment:
CC: @CharlieFRuan @davidpissarra I updated a few pieces in the code to make sure `prefill_chunk_size` is upper-bounded by `sliding_window_size` and `context_window_size`